src/evergreen/performance_results.py

```diff
@@ -35,16 +36,11 @@ def __init__(self, test_result, api):
             test_result.get('start', test_result['results']['start']))
         self.end = parse_evergreen_datetime(
             test_result.get('end', test_result['results']['end']))
-        # Microbenchmarks does not produce a 'workload' field. We need to fill in the 'workload'
-        # field for microbenchmark points in order to query on 'workload'.
-        self.maximum_thread_level, self.maximum_operations_per_second = _get_max_ops_per_sec(
-            test_result)
 
     @property
     def test_results(self):
-        if self.maximum_operations_per_second is None:
-            return None
-        return _get_performance_results(self.json, self._api)
+        return [PerformanceTestResult(item, self._api) for item in
```

Comment:
[nit] I'm not a huge fan of a line break in the middle of the `for` statement. I find the following easier to read:
```
return [PerformanceTestResult(item, self._api)
            for item in _format_performance_results(self.json['results'])]
```